### ISSUE TITLE
can I use this w replit and uptimer bot?

### ISSUE THREAD

**Issue by AnimeKally** [BridgeSenseDev/Dank-Memer-Grinder#240]
**Describe the bug**
A clear and concise description of what the bug is and steps to reproduce.

**Error**
Click on the home tab and send the full error if there is one.

**Screenshots**

**OS:**
Windows, Linux arm64, Linux amd64 or MacOS

**Comment by EchoQuill** [comment]
Nope, since this is gui. Also replit blocks selfbots now if iam right, so even if it wasn't a gui then also it wouldn't work. 
[keep in mind that this is being re-written and a working version has not been released in releases yet.]

**Comment by VillainsRule** [comment]
there is a cli option for mac, no?

**Comment by BridgeSenseDev** [comment]
there is cli option, https://docs.dankmemer.tools/installation/pre-built-binaries
